.github/workflows/ci.yaml

```diff
@@ -0,0 +1,46 @@
+name: ci
+
+on:
+  pull_request:
+    branches: '*'
```

Comment:
I think this needs to be `**` because the branch name includes `/`:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
>*: Matches zero or more characters, but does not match the / character.

Or I guess you can just omit it.

Reply:
@yuya-takeyama
Thank you. I'll fix it.
This change is actually based on the work of @tagomoris. https://github.com/woothee/woothee-js/pull/21